spec/models/user_spec.rb

```diff
@@ -12,16 +15,21 @@
     expect(user).to_not be_valid
     expect(user.errors[:password_confirmation]).to_not be_blank
   end
-end
 
-RSpec.describe 'Confirm admin role' do
-  it 'is not an admin if the role is not admin' do
-    user = FactoryGirl.create(:user)
-    expect(user.admin?).to eq(false)
+  describe 'Confirm username' do
+    it 'should have a username' do
+      expect(user.username).to eq(username)
+    end
   end
 
-  it 'is an admin if the role is admin' do
-    user = FactoryGirl.create(:user, role: 'admin')
-    expect(user.admin?).to eq(true)
+  describe 'Confirm admin role' do
+    it 'is not an admin if the role is not admin' do
```

Comment:
So what you're saying here in the title is that a user object is not an admin if its role is not admin, but what you're testing is #admin? returns false if user is not an admin. I would rethink the naming of this test to be more explicit